cms/djangoapps/contentstore/views/preview.py

```diff
@@ -241,7 +216,6 @@ def _preview_module_system(request, descriptor, field_data):
         services={
             "field-data": field_data,
             "i18n": ModuleI18nService,
-            "library_tools": LibraryToolsService(modulestore()),
```

Comment:
It was (and is) but since the service is added [here at the CachingDescriptorSystem level](https://github.com/edx/edx-platform/blob/982e99b/common/lib/xmodule/xmodule/modulestore/split_mongo/caching_descriptor_system.py#L85), it was redundant to define it at the `PreviewModuleSystem` level as well.